src/main/resources/lab_setup.sql

```diff
@@ -0,0 +1,8 @@
+DROP TABLE IF EXISTS member;

```

Comment:
Could/should we configure the Spring Boot app to run this file on start up?

Reply:
Hmm... I could change it to data.sql to have it run on start up to eliminate some pre-work! Good idea!